test/e2e/utils.go

```diff
@@ -574,28 +575,22 @@ func wasUsingOtelCollector(jaegerInstanceName, namespace string) bool {
 	return false
 }
 
-// Returns a collector spec which uses the Open Telementry Collector
-func getOtelCollectorSpec(collectorSpec v1.JaegerCollectorSpec) v1.JaegerCollectorSpec {
-	otelOptions := make(map[string]interface{})
-
-	processors := map[string]string{"queued_retry": ""}
-	otelOptions["processors"] = processors
-
-	port := map[string]string{"port": "14269"}
-	healthCheck := map[string]interface{}{"health_check": port}
-	otelOptions["extensions"] = healthCheck
-
-	traces := map[string]string{"processors": "queued_retry"}
-	pipelines := map[string]interface{}{"traces": traces}
-	service := map[string]interface{}{"pipelines": pipelines}
-
-	otelOptions["service"] = service
-
-	otelCollectorSpec := &v1.JaegerCollectorSpec{
-		Image:   "jaegertracing/jaeger-opentelemetry-collector:latest",
-		Config:  v1.NewFreeForm(otelOptions),
-		Options: collectorSpec.Options,
+func getOtelCollectorOptions() map[string]interface{} {
+	otelOptions := map[string]interface{}{
+		"processors": map[string]string{
+			"queued_retry": "",
+		},
+		"extensions": map[string]interface{}{
+			"health_check": map[string]string{"port": "14269"},
+		},
+		"service": map[string]interface{}{
+			"pipelines": map[string]interface{}{
+				"traces": map[string]string{
+					"processors": "queued_retry",
```

Comment:
Why do we need `queued_retry` processor?

Reply:
I have no idea.  I just copied the example from your PR.  :-)  

Which lines should I remove?

Reply:
The whole service can be removed. We just need to override the HC port.